apps/web/src/styles/features/settings/foundation.css

```
@@ -0,0 +1,1786 @@
+  min-height: 420px;
+  margin-top: 14px;
+}
+
+@media (max-width: 820px) {
+  .courses-main.student-surface-main.courses-main--settings {
+    padding-inline: 15px;
+  }
+
+  .student-surface-main > .settings-page {
+    --settings-page-inline-gutter: 0px;
+    padding: 0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)